tests/features/page-header.feature

```diff
@@ -13,13 +13,25 @@ Feature: Page header block component.
     When I am an anonymous user
     And I go to the "Robots are everywhere" demo page
     Then I should see the heading "Robots are everywhere" in the "page header"
+    And the breadcrumb should contain the following items:
```

Comment:
And the breadcrumb trail should be "Home"
And the breadcrumb active element should be "Robots are everywhere"

Reply:
In case of multiple links, it would be something like:
`And the breadcrumb trail should be "Home, Term 1, Term 1-1"`